README.md

```diff
@@ -1,52 +1,109 @@
 # Symbl Personal Assistant Meeting App
```

Comment:
Please add respective covenants.

Reply:
@eric-giannini covenant added in latest commit